sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1451,6 +1451,15 @@ object SQLConf {
     .intConf
     .checkValues((1 to 9).toSet + Deflater.DEFAULT_COMPRESSION)
     .createWithDefault(Deflater.DEFAULT_COMPRESSION)
+
+  val SETOPS_PRECEDENCE_ENFORCED =
+    buildConf("spark.sql.setops.precedence.enforced")
```

Comment:
let me think about the name of conf

Reply:
@gatorsmile Sure.

Reply:
@gatorsmile A couple of alternatives i had considered :-). FYI.
`spark.sql.set.operators.precedence.enforced`
`spark.sql.set.operators.standard.compliance`